src/main/kotlin/com/cognifide/gradle/aem/environment/docker/Container.kt

```diff
@@ -0,0 +1,54 @@
+package com.cognifide.gradle.aem.environment.docker
+
+import com.cognifide.gradle.aem.environment.EnvironmentException
+import org.buildobjects.process.ExternalProcessFailureException
+import org.buildobjects.process.ProcBuilder
+
+class Container(val name: String) {
+
+    fun isRunning(): Boolean {
+        val containerId = containerId()
+        if (containerId == null) {
+            return false
+        }
+        try {
+            return ProcBuilder("docker")
```

Comment:
Gradle has API for executing things

e.g I am executing AEM quickstart to unpack like this: https://github.com/Cognifide/gradle-aem-plugin/blob/master/src/main/kotlin/com/cognifide/gradle/aem/instance/LocalInstance.kt#L209

there is also `project.exec {}` / https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#exec-org.gradle.api.Action-

consider using it if possible